PHPCSStandards/PHPCSUtils
# Release 1.0.0-alpha3

## Release checklist

### Update website
- [x] Regenerate the PHPDoc documentation - PR #156
    :pencil2: Use phpDocumentor 3 with the command `phpdoc --visibility=public,protected`
- [x] Sync any changes in the Readme into the website `index.md` file. - PR #157
    :pencil2: Copy & pasting the content from the `README.md` file to `docs/index.md` (and double-check the few remaining differences are intact).
    To verify the output locally:
    ```bash
    bundle update
    bundle exec jekyll serve
    ```
    and then visiting http://localhost:4000/ to see the result.

### General
- [x] Update the `DEVMASTER` version nr constant in the `Tests/BackCompat/Helper/GetVersionTest.php` file. - PR #140
- [x] Verify, and if necessary, update the allowed version ranges for various dependencies in the `composer.json` - PR #161, #162
- [x] Add changelog for the release - PR #158
    :pencil2: Remember to add a release link at the bottom and to adjust the link for "Unreleased"!

### Release
- [x] Add release tag against `master` (careful, GH defaults to `develop`!) & copy & paste the changelog to it
    :pencil2: Don't forget to copy the link collection from the bottom of the changelog!
- [x] Close the milestone
- [x] Open a new milestone for the next release
- [x] If any open PRs/issues which were milestoned for this release did not make it into the release, update their milestone.
- [ ] Tweet about the release.
- [x] Fast-forward `develop` to be equal to `master`
